Provisioners/Nginx/Debian/Requirements

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exit 0
```

Comment:
I don't understand the end of your message but this is something I was thinking about. Maybe we can avoid running the requirements or other elements of the Provider Interface by hand, and force the use of a command. This command would source the prelude itself. Thoughts?